pkg/configapi/v1/types.go

```diff
@@ -0,0 +1,336 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/resource"
+	"k8s.io/kubernetes/pkg/util"
+)
+
+type KubeletConfiguration struct {
+	api.TypeMeta
```

Comment:
Kubeconfig just inlines the kind and apiVersion fields:
https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/clientcmd/api/v1/types.go

The scheduler (incorrectly) references the v1 api TypeMeta:
https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/scheduler/api/v1/types.go

Neither of these uses ObjectMeta.

Just copy TypeMeta or use the unversioned instance, if that exists.